### DISCUSSION TITLE
Obtaining subscription summary

### DISCUSSION THREAD

**Discussion by nunga** [DIGITALCRIMINALS/OnlyFans#190] [Ideas]
When I go to [https://onlyfans.com/my/subscriptions/active](https://onlyfans.com/my/subscriptions/active) I see my 138 subscriptions (many are free).

Some will auto-renew, others won't. OnlyFans doesn't provide this in a neat table that I can quickly skim to see what's expiring and what will be auto-renewing. Is this information available when listing subscriptions?

It'd be really useful to see the list updated to show:

Scraping Paid Content
Scraping - modelname | 1 / 138 | expires yyyy-mm-dd
Scraping - modelname | 2 / 138 | free
Scraping - modelname | 3 / 138 | renews yyyy-mm-dd
Scraping - modelname | 4 / 138 | expires yyyy-mm-dd

Thanks.

**Comment by DIGITALCRIMINAL** [comment]
Yeah, I'm working on the ux experience w/ https://github.com/willmcgugan/rich

So you'll have a dashboard, but it's not ready yet
